src/style.css

```
@@ -0,0 +1,110 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 30px 14px 14px;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.card__photo img {
+  width: 100%;
+  height: 100%;
+}
+
+.card__description {
+  margin-bottom: 15px;
+}
+
+.card__title {
+  margin: 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  height: 31px;
+}
+
+.card__code {
+  margin: 4px 0 0;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.star_item {
+  width: 15px;
+  height: 15px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star_item:not(:last-child) {
+  margin-right: 4px;
+}
+
+.star_item:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__price {
+  margin: 20px 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__price_amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__buybutton {
```

```suggestion
.card__buy-button {
```
если название элемента состоит из двух слов - они пишутся через дефис